File: 1108.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 0 ms, faster than 100.00% of C++ online submissions
	Memory Usage: 6.1 MB, less than 31.33% of C++ online submissions
*/

class Solution {
public:
    string defangIPaddr(string address) {
        string ans="";
        for(char c: address){
            if(c=='.'){
                ans.push_back('[');
                ans.push_back('.');
                ans.push_back(']');
            }else{
                ans+=c;
            }
        }
        return ans;
    }
};

File: 1108.py
# Author: Zoljargal Gantumur
# Runtime: 28 ms, faster than 73.30% of Python3 online submissions
# Memory Usage: 13.8 MB, less than 71.99% of Python3 online submissions

class Solution:
    def defangIPaddr(self, address: str) -> str:
        return "[.]".join(address.split('.'))
        
